packages/status-js/src/client/storage.ts

```diff
@@ -0,0 +1,53 @@
+type Unsubscribe = () => void
+
+export interface Storage {
+  getItem: <T = string>(key: string) => T | null
+  setItem: (key: string, newValue: unknown) => void
+  removeItem: (key: string) => void
+  subscribe?: (key: string, callback: (value: string) => void) => Unsubscribe
+}
+
+export class LocalStorage implements Storage {
+  #prefix = 'status'
```

Comment:
Shouldn't the prefix include Community's pub key in case the parent page embeds it under a diff key, changes it, or, probably less likely, embeds more than one?

Reply:
Isn't it worth validating account's pub key against the member list first?

Reply:
https://github.com/status-im/status-web/issues/317